estate/models/EstateProperty.py

```diff
@@ -0,0 +1,44 @@
+from datetime import datetime
+from odoo import models, fields
+
+class EstateProperty(models.Model):
```

Comment:
```suggestion


class EstateProperty(models.Model):
```
Two space new line before defining a class